File: src/main/java/com/ject/studytrip/pomodoro/infra/jpa/PomodoroJpaRepository.java
package com.ject.studytrip.pomodoro.infra.jpa;

import com.ject.studytrip.pomodoro.domain.model.Pomodoro;
import java.util.Optional;
import org.springframework.data.jpa.repository.JpaRepository;

public interface PomodoroJpaRepository extends JpaRepository<Pomodoro, Long> {

    Optional<Pomodoro> findByDailyGoalId(Long dailyGoalId);
}

File: src/main/kotlin/com/ject/studytrip/pomodoro/infra/jpa/PomodoroJpaRepository.kt
package com.ject.studytrip.pomodoro.infra.jpa

import com.ject.studytrip.pomodoro.domain.model.Pomodoro
import org.springframework.data.jpa.repository.JpaRepository
import java.util.Optional

interface PomodoroJpaRepository : JpaRepository<Pomodoro, Long> {
    fun findByDailyGoalId(dailyGoalId: Long): Optional<Pomodoro>
}
